src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -372,7 +377,10 @@ MulticopterAttitudeControl::MulticopterAttitudeControl() :
 	_saturation_status{},
 	/* performance counters */
 	_loop_perf(perf_alloc(PC_ELAPSED, "mc_att_control")),
-	_controller_latency_perf(perf_alloc_once(PC_ELAPSED, "ctrl_latency"))
+	_controller_latency_perf(perf_alloc_once(PC_ELAPSED, "ctrl_latency")),
+	_foaw_rollrate(0.004f, 0.025f), // FOWA differentiator (dt, noise amplitude)
```

Comment:
couldn't dt be the measured time instead of hard coded? because otherwise it breaks as soon as we change the update frequency.

Reply:
This is just an initialization value. The sampling time is updated at each update using `set_sample_time(dt)` (lines 1013 - 1015)